app/routes/device.py

```diff
@@ -70,11 +70,25 @@ def update_device(deviceId):
     updated_row = Database.update_device(deviceId, **body)
 
     if updated_row is None:
+        Database.write_log(
+            role      = g.tokenPayload["role"],
+            action    = f"User {g.tokenPayload['user_id']} tried updating unknown device {deviceId}",
```

Comment:
might be better to write the username as well
```suggestion
            action    = f"User {g.tokenPayload['user_id']} {g.tokenPayload['username']} tried updating unknown device {deviceId}",
```